packages/core/src/bag3d/core/assets/ahn/core.py

```diff
@@ -142,7 +142,7 @@ def download_ahn_index_esri(ahn_version: int, with_geom: bool = False) -> Union[
     return features
 
 
-def tile_index_origin() -> Tuple[float, float, float, float]:
+def tile_index_origin() -> Tuple[float, float, float, float]: # pragma: no cover
```

Comment:
this function seems to be unused in the whole repo - so I removed it from the coverage. Should it be removed?